### ISSUE TITLE
ServerIncompatibilityError with version 5.20.0

### ISSUE THREAD

**Issue by schlegelp** [CAVEconnectome/CAVEclient#178]
With `cave-client` version `5.20.0` I get a new `ServerIncompatibilityError`:

```python
>>> client = cv.CAVEclient('flywire_fafb_public')
>>> client.chunkedgraph.level2_chunk_graph(720575940614131061)
ServerIncompatibilityError: Use of keyword argument `bounds` in `level2_chunk_graph` is only permitted for
server version >=2.15.0, your server version is None. Contact your system
administrator to update the server version.
```

Same code works like a charm with `5.18.0`. I assume this is a configuration issue?

**Comment by bdpedigo** [comment]
hi @schlegelp - this is related to a new scheme we implemented for comparing the compatibility of the client and the version of the service running remotely, but this shouldn't be erroring here. will look into it!

**Comment by ceesem** [comment]
We have a new system in place that should allow features to be introduced to caveclient but not passed along to a server deployment if it is not yet compatible. It apparently is too aggressive — I propose a fix in #179.
